apps/04TowersOfHanoi.js

```diff
@@ -43,6 +45,7 @@ function checkForWin() {
 
```

Comment:
Line 34 above:

```javascript
     else {
        return startStackArray[startStackArray.length -1] < endStackArray [endStackArray.length -1];
     }
```
does not seem correct. This statement should be the logic in an if, then either a return true or return false should happen.